service/src/main/java/com/ericsson/eiffel/remrem/generate/App.java

```diff
@@ -25,8 +25,8 @@
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.PropertySource;
 
-@SpringBootApplication
-@ComponentScan("com.ericsson.eiffel.remrem")
+@SpringBootApplication(scanBasePackages = {"com.ericsson.eiffel.remrem", "com.ericsson.eiffel.remrem.semantics"})
+@ComponentScan({"com.ericsson.eiffel.remrem", "com.ericsson.eiffel.remrem.semantics"})
```

Comment:
The @SpringBootApplication annotation already includes the @ComponentScan annotation. So I believe there is no specific need to add the @ComponentScan annotation separately as the same packages are mentioned with scanBasePackages for the @SpringBootApplication annotation.

Reply:
You're right. I was unable to get it working unless both of them were presented. It works fine now with @SpringBootApplication only. Removing @ComponentScan.